### PR TITLE
DOCS-10255-AddingArchitecureDrawing-ESB

### PR DESCRIPTION
**Pull request by emileemooreVT**

### CODE REVIEW

File: modules/ROOT/pages/partner-manager-architecture.adoc [hunk 1]
```diff
@@ -1,2 +1,62 @@
...
+[%header%autowidth.spread]
+|===
+|Callout |Name |Description
+|image:pm-arch-diagram-1.png[]
+|Partner Manager
+|MuleSoft hosted cloud control plane allowing users to configure trading partner profiles, build and deploy B2B integration message flows, and monitor B2B transactions activity.
+
+|image:pm-arch-diagram-2.png[]
+|Receive Endpoint Services
+|Service to receive inbound B2B or EDI messages from partners via communication protocols such as AS2 and SFTP or outbound application messages from your organization’s backend systems via HTTP, HTTPS, or SFTP.
```
Comment (vijayang) [inline modules/ROOT/pages/partner-manager-architecture.adoc]: Please rephrase on the below lines:

Service to receive inbound B2B or EDI messages from partners via transport protocols such as AS2/SFTP, or outbound application messages from your organization’s backend systems via HTTP(s)/SFTP.

File: modules/ROOT/pages/partner-manager-architecture.adoc [hunk 2]
```diff
@@ -1,2 +1,62 @@
...
+|Document Processing Service
+|Core B2B processing engine responsible for parsing, validating and translating received B2B messages based on configurations and mapping in Partner Manager.
+
+|image:pm-arch-diagram-4.png[]
+|API Replication Service
+|Tracking service responsible for updating transmissions and transaction activities to the control plane and to the payload storage API.
+
+|image:pm-arch-diagram-5.png[]
+|Send Endpoint Services
+|Service to deliver translated inbound application messages to internal applications such as ERP, billing, Order Management via HTTP, HTTPS or SFTP; or outbound B2B/EDI payloads to your trading partners via AS2, HTTP, HTTPS or SFTP.
```
Comment (vijayang) [inline modules/ROOT/pages/partner-manager-architecture.adoc]: Please rephrase on the below lines:
Service to deliver translated inbound application messages to internal applications such as ERP, billing, Order Management via transport protocols like HTTP(S)/SFTP; or outbound B2B/EDI payloads to your trading partners via AS2/HTTP(s)/SFTP.

**Review by vijayang — CHANGES_REQUESTED** [review]
Looks good. Requesting couple of minor changes.

File: modules/ROOT/pages/partner-manager-architecture.adoc [hunk 3]
```diff
@@ -1,2 +1,63 @@
 = Anypoint Partner Manager Architecture
 
+With Anypoint Partner Manager, you can deploy B2B Integration message flows with your partners, either to your own hosted Mule runtimes or to MuleSoft hosted CloudHub iPaaS platform.
```
Comment (kbostrom) [inline modules/ROOT/pages/partner-manager-architecture.adoc]: Why capitalize Integration?

Comment (kbostrom) [reply modules/ROOT/pages/partner-manager-architecture.adoc]:
```suggestion
With Anypoint Partner Manager, you can deploy B2B integration message flows with your partners, either to your own hosted Mule instances or to a MuleSoft-hosted CloudHub iPaaS platform.
```

Comment (kbostrom) [reply modules/ROOT/pages/partner-manager-architecture.adoc]: GLOBAL re. "B2B integration"

File: modules/ROOT/pages/partner-manager-architecture.adoc [hunk 4]
```diff
@@ -1,2 +1,63 @@
...
+
+The functional features and user experience in configuring B2B Integrations with Partner Manager are deployment-target agnostic; however, there are differences in the underlying architecture when implementing the functionality in the runtime.
```
Comment (kbostrom) [inline modules/ROOT/pages/partner-manager-architecture.adoc]: Does "the runtime" refer to the Mule runtime engine? 
Do we mean "in Mule" or "in the Mule instance"?
GLOBALs

File: modules/ROOT/pages/partner-manager-architecture.adoc [hunk 5]
```diff
@@ -1,2 +1,63 @@
...
+
+When you deploy a configured message flow, Partner Manager automatically translates the configurations used in the message flow and deploys and updates a set of applications necessary to orchestrate end-to-end B2B processing to the Mule runtime. These applications include:
```
Comment (kbostrom) [inline modules/ROOT/pages/partner-manager-architecture.adoc]: to the Mule runtime engine (to Mule, to the Mule instance)

File: modules/ROOT/pages/partner-manager-architecture.adoc [hunk 6]
```diff
@@ -1,2 +1,63 @@
...
+* Send endpoint services
+
+image::partner-manager-high-level-arch.png[Anypoint Partner Manager Architecture]
+
+[%header%autowidth.spread]
+|===
+|Callout |Name |Description
+|image:pm-arch-diagram-1.png[]
+|Partner Manager
+|MuleSoft hosted cloud control plane allowing users to configure trading partner profiles, build and deploy B2B integration message flows, and monitor B2B transactions activity.
```
Comment (kbostrom) [inline modules/ROOT/pages/partner-manager-architecture.adoc]:
```suggestion
|MuleSoft-hosted cloud control plane enabling users to configure trading partner profiles, build and deploy B2B integration message flows, and monitor B2B transaction activity
```

File: modules/ROOT/pages/partner-manager-architecture.adoc [hunk 7]
```diff
@@ -1,2 +1,63 @@
...
+
+|image:pm-arch-diagram-2.png[]
+|Receive Endpoint Services
+|Service to receive inbound B2B or EDI messages from partners transport protocols such as AS2 and SFTP or outbound application messages from your organization’s backend systems via HTTP, HTTPS, or SFTP.
```
Comment (kbostrom) [inline modules/ROOT/pages/partner-manager-architecture.adoc]: Service or Services?

Comment (kbostrom) [reply modules/ROOT/pages/partner-manager-architecture.adoc]:
```suggestion
|Services to receive inbound B2B or EDI messages from partner transport protocols such as AS2 and SFTP or outbound application messages from your organization’s backend systems via HTTP, HTTPS, or SFTP
```

File: modules/ROOT/pages/partner-manager-architecture.adoc [hunk 8]
```diff
@@ -1,2 +1,63 @@
...
+
+|image:pm-arch-diagram-3.png[]
+|Document Processing Service
+|Core B2B processing engine responsible for parsing, validating and translating received B2B messages based on configurations and mapping in Partner Manager.
```
Comment (kbostrom) [inline modules/ROOT/pages/partner-manager-architecture.adoc]:
```suggestion
|Core B2B processing engine responsible for parsing, validating, and translating received B2B messages based on configurations and mapping in Partner Manager
```

File: modules/ROOT/pages/partner-manager-architecture.adoc [hunk 9]
```diff
@@ -1,2 +1,63 @@
...
+
+|image:pm-arch-diagram-4.png[]
+|API Replication Service
+|Tracking service responsible for updating transmissions and transaction activities to the control plane and to the payload storage API.
```
Comment (kbostrom) [inline modules/ROOT/pages/partner-manager-architecture.adoc]: no period

File: modules/ROOT/pages/partner-manager-architecture.adoc [hunk 10]
```diff
@@ -1,2 +1,63 @@
...
+
+|image:pm-arch-diagram-5.png[]
+|Send Endpoint Services
+|Service to deliver translated inbound application messages to internal applications such as ERP, billing, Order Management via HTTP, HTTPS or SFTP; or outbound B2B/EDI payloads to your trading partners via AS2, HTTP, HTTPS or SFTP.
```
Comment (kbostrom) [inline modules/ROOT/pages/partner-manager-architecture.adoc]: Service or Services?

Comment (kbostrom) [reply modules/ROOT/pages/partner-manager-architecture.adoc]: (I'll let you reword per tech reviewer input....)

File: modules/ROOT/pages/partner-manager-architecture.adoc [hunk 11]
```diff
@@ -1,2 +1,63 @@
...
+
+|image:pm-arch-diagram-6.png[]
+|Payload Storage API
+|API to store B2B payload data in your own storage solution.
```
Comment (kbostrom) [inline modules/ROOT/pages/partner-manager-architecture.adoc]: no period

The ideal is that list item descriptions are parallel (either all fragments or all periods, without and with periods, respectively). If your descriptions aren't parallel, and at least one includes a complete sentence, then all must end with periods. But it's best to use parallel formats.

File: modules/ROOT/pages/partner-manager-architecture.adoc [hunk 12]
```diff
@@ -1,2 +1,63 @@
...
+|===
+
+== Partner Manager for CloudHub
+
+When deploying B2B Integration message flows to CloudHub, Anypoint MQ queues are used between processing stages to add additional reliability to the B2B message processing.
```
Comment (kbostrom) [inline modules/ROOT/pages/partner-manager-architecture.adoc]: Try to reword to use active voice.

File: modules/ROOT/pages/partner-manager-architecture.adoc [hunk 13]
```diff
@@ -1,2 +1,63 @@
...
+
+As of the time of first deployment from Partner Manager, runtime applications are deployed by default to the CloudHub region specified in the business group’s default settings in Runtime Manager. Anypoint MQ queues are automatically created during the deployment process in the same or the nearest available region. Any new runtime application from subsequent deployments is deployed to the same region as the first deployment.
```
Comment (kbostrom) [inline modules/ROOT/pages/partner-manager-architecture.adoc]: Also passive.

File: modules/ROOT/pages/partner-manager-architecture.adoc [hunk 14]
```diff
@@ -1,2 +1,63 @@
...
+
+image::partner-manager-cloud-deployment.png[Anypoint Partner Manager for CloudHub]
+
+== Partner Manager for On-Premise Mule Runtime
```
Comment (kbostrom) [inline modules/ROOT/pages/partner-manager-architecture.adoc]:
```suggestion
== Partner Manager for On-Premise Mule Instances
```

File: modules/ROOT/pages/partner-manager-architecture.adoc [hunk 15]
```diff
@@ -1,2 +1,63 @@
...
+
+When deploying B2B Integration message flows to your own hosted on-premise Mule runtime, HTTP calls are used between processing stages during B2B message processing.
```
Comment (kbostrom) [inline modules/ROOT/pages/partner-manager-architecture.adoc]:
```suggestion
When deploying B2B Integration message flows to your own hosted on-premise Mule instance, HTTP calls are used between processing stages during B2B message processing.
```

Comment (kbostrom) [reply modules/ROOT/pages/partner-manager-architecture.adoc]: ...also passive voice. 
Review and try to reword to active voice GLOBALLY.

File: modules/ROOT/pages/partner-manager-architecture.adoc [hunk 16]
```diff
@@ -1,2 +1,63 @@
...
+
+The runtime applications are deployed to the server group (B2BServerGroup) that you will have configured in your data center or your private cloud.
```
Comment (kbostrom) [inline modules/ROOT/pages/partner-manager-architecture.adoc]: passive voice > active

Comment (kbostrom) [reply modules/ROOT/pages/partner-manager-architecture.adoc]: "will" > present tense

Comment (kbostrom) [reply modules/ROOT/pages/partner-manager-architecture.adoc]: I am currently checking in with Fer, Hanna, and Duke about the subtleties around using "runtime," which have never been crystal clear to me. If referring to the engine, I know we want to use one of Mule runtime engine (Mule) or refer to the on-prem "instance." 
I'll share what I find out; please feel free to correct/share with me!

Comment (kbostrom) [reply modules/ROOT/pages/partner-manager-architecture.adoc]: P.S. Don't let this slow you down.

**Review by kbostrom — DISMISSED** [review]
CE

File: modules/ROOT/pages/partner-manager-architecture.adoc (hunk 4 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/partner-manager-architecture.adoc]: In some cases, when they say "in the runtime", that is the phrasing used in other parts of the doc. For instances where it says "Mule runtime" that can be changed to "Mule instance"

File: modules/ROOT/pages/partner-manager-architecture.adoc (hunk 9 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/partner-manager-architecture.adoc]: Done!

File: modules/ROOT/pages/partner-manager-architecture.adoc (hunk 2 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/partner-manager-architecture.adoc]: Done!

File: modules/ROOT/pages/partner-manager-architecture.adoc (hunk 10 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/partner-manager-architecture.adoc]: Services!

File: modules/ROOT/pages/partner-manager-architecture.adoc (hunk 11 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/partner-manager-architecture.adoc]: Got it!

File: modules/ROOT/pages/partner-manager-architecture.adoc (hunk 12 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/partner-manager-architecture.adoc]: Done!

File: modules/ROOT/pages/partner-manager-architecture.adoc (hunk 16 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/partner-manager-architecture.adoc]: Done!

**Review by kbostrom — APPROVED** [review]
RA

**Review by vijayang — APPROVED** [review]

**Pull request merged**